try {
  console.log(3 - 2);
  throw "I AM AN EXCEPTION/";
} catch (error) {
  console.log(error);
}

console.log("YO");
